operator/roles/csiscaleoperator/defaults/main.yml

```diff
@@ -56,3 +56,5 @@ state: present
 
 # Kubelet root directory path.
 kubeletRootDirPath: "{{ _csi_ibm_com_csiscaleoperator.spec.kubeletRootDirPath | default('/var/lib/kubelet') }}"
+
+shortnameNodeMapping: "{{ lookup('env', 'SHORTNAME_NODE_MAPPING') | default('no') }}"
```

Comment:
for some reason default part is not working. when env is not specified in operator.yaml then remain empty in driver pod. I tried following version which I found working
_shortnameNodeMapping: "{{ lookup('env', 'SHORTNAME_NODE_MAPPING') | default('no', true) }}"_